serverless/packages/functions/src/dbutils/table_exists.ts

```diff
@@ -0,0 +1,6 @@
+import { get_all_tablenames } from "./get_all_tablenames";
+
+export async function table_exists(table_name: string) : Promise<boolean> {
+    const tableNames: string[] = await get_all_tablenames();
+    return tableNames.indexOf(table_name) !== -1;
+}
```

Comment:
Minor: can you please run Prettier for all changed files? There are some aesthetic issues; for example, each file should end with a blank line.

We'll need to update our checks to also check /serverless in addition to /frontend